src/Squirrel/UpdateManager.ApplyReleases.cs

```diff
@@ -68,6 +73,7 @@ public async Task<string> ApplyReleases(UpdateInfo updateInfo, bool silentInstal
                     "Failed to invoke post-install");
 
                 progress(95);
+                status("Preparing the update");
```

Comment:
This is quite confusing for me. Looks to me like we are just going to install update... Please, delete it or change the status. Might be "Cleaning old files" or anything else.

Reply:
I will change that to "Cleaning old files" because I'm suppose (but not sure) the clean-up may take some time and it is better to update the previous status to something newer.